src/playercards/cards/ScrollofSecrets.ttslua

```diff
@@ -0,0 +1,92 @@
+-- this script is shared between the lvl 0 and lvl 3 versions of Scroll of Secrets
+local mythosAreaApi = require("core/MythosAreaApi")
+local playmatApi    = require("playermat/PlaymatApi")
+
+-- get class via metadata and create context menu accordingly
+function onLoad()
+  local notes = JSON.decode(self.getGMNotes())
+  if notes then
+    createContextMenu(notes.class)
+  else
+    print("Missing metadata for Scroll of Secrets!")
+  end
+end
+
+function createContextMenu(class)
+  local colorList = Player.getAvailableColors()
+
+  if class == "Seeker|Mystic" then
+    -- draw 1 card from the bottom
+    self.addContextMenuItem("Draw bottom card", function(playerColor) contextFunc(playerColor, 1) end)
+  elseif class == "Seeker" then
+    -- draw 3 cards from the bottom
+    self.addContextMenuItem("Draw bottom card(s)", function(playerColor) contextFunc(playerColor, 3) end)
+  elseif class == "Mystic" then
+    -- draw 1 card from the bottom
+    self.addContextMenuItem("Draw bottom card", function(playerColor) contextFunc(playerColor, 1) end)
+  end
+end
+
+function contextFunc(playerColor, amount)
+  -- TODO: only show colors that have a deck
+  local options = Player.getAvailableColors()
+  table.insert(options, "Encounter Deck")
+
+  -- show the target selection dialog
+  Player[playerColor].showOptionsDialog("Select target deck", options, _, 
+    function(owner)
+      drawCardsFromBottom(playerColor, owner, amount)
+    end
+  )
+end
+
+function drawCardsFromBottom(playerColor, owner, amount)
```

Comment:
Are we sure we want to be _drawing_ the cards?  Since this is a look operation that can result in the card going to any of 4 places it seems the least likely place for a card to end up is the hand of the player activating the ability.  I think this may be better envisioned as a camera refocus and let players manually draw the card and choose where it goes.  But honestly at that point, I question if it's even worth adding the feature at all...?

Reply:
We could add a broadcast statement to make this clear, but indeed the shoe point is simplifying drawing cards from the bottom.

Reply:
Added an additional print instruction on how to proceed with the drawn cards.